neon_stt_plugin_coqui/__init__.py

```diff
@@ -100,7 +146,17 @@ def convert_samplerate(self, audio, desired_sample_rate):
         return desired_sample_rate, np.frombuffer(output, np.int16)
 
 
-    def execute(self, audio, language=None):
+    def execute(self, audio):
+        '''
+        Executes speach recognition
+        Reads audio from file path
+
+        Parameters:
+                    audio (str): path to audio file
```

Comment:
I missed this in an earlier PR, but `audio` here should be an `AudioData` object, rather than a file path (https://github.com/OpenVoiceOS/ovos-core/blob/a8c251b3669e5d025855d9fceaff688998f7c23e/mycroft/listener/mic.py#L716)

Consider adding an extra method that accepts file paths if it is helpful for backwards-compat. or updating references to pass `AudioData` instead of file paths. There's an example of how to convert a file into `AudioData` here: https://github.com/NeonGeckoCom/neon_speech/blob/5df32633c5a19337c5222fe2c3cf0c24cd84d8e2/neon_speech/service.py#L258

Reply:
I tried to change that, but I can not apply model.stt() on AudioData object as it expects int array - a 16-bit, mono raw audio signal. I also tried AudioData.get_raw_data() and AudioData.get_wav_data() but this methods return bytes data only

Reply:
Maybe this example in the Scribosermo plugin (or part of it) will help: https://github.com/NeonGeckoCom/neon-stt-plugin-scribosermo/blob/083732e11cfd68a7937b423a403524986512f77b/neon_stt_plugin_scribosermo/__init__.py#L85

Reply:
Yes, that helped